index.js

```diff
@@ -65,9 +66,17 @@ module.exports = async (target, options) => {
 			cliArguments.push('--background');
 		}
 
+		if (options.newWindow) {
+			cliArguments.push('-n');
+		}
+
 		if (app) {
 			cliArguments.push('-a', app);
 		}
+
+		if (appArguments.length > 0) {
+			cliArguments.push('--args', ...appArguments);
+		}
```

Comment:
Are you sure moving it here is correct? I think the `target` (https://github.com/sindresorhus/open/pull/206/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R155) is order dependent.